app/docs/reference/compatible-devices.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Compatible devices | Reference"
+description: "Lists devices compatible with Routegy QR codes and NFC tags"
```

Comment:
I've been trying to form these as questions relative to their section. For reference, these are "how does X work".

I think for this, i'd recommend: `How do I know if my device is compatible?`